src/app/workflow/workflow.component.html

```diff
@@ -96,6 +96,15 @@ <h3>
           >
             View Public
           </a>
+          <button 
+            mat-flat-button
+            id="DOIButton"
+            class="mr-2"
+            color="accent"
+           (click)="requestDOIForWorkflowVersion()"
+           [disabled]="(isRefreshing$ | async) || selectedVersion.doiURL"
```

Comment:
I think this should be `selectedVersion?doiURL`.

Reply:
selectedVersion?doiURL was not accepted when I tried it

Reply:
Sorry, `selectedVersion?.doiUrl`, which means it won't fail if selectedVersion is null/undefined.

Reply:
done